frontend/src/components/editor/output/Outputs.css

```diff
@@ -7,8 +7,9 @@
   width: 100%;
 }
 
-/* Just applied to vertical layout */
-.output-area .output {
+/* No margin in edit mode, where we already have separation
+ * between cells. */
```

Comment:
Snuck this in to fit more content on the page in edit mode. Realized we only need these margins in app mode.

Reply:
yep agreed!